### PR TITLE
Reverting to official discord.py library

### PR DESCRIPTION
**Pull request by jammiess**
## Description

Revert to official discord.py library.

## Checklist

- [X] [Linked PR to corresponding issue in this repo](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
- [X] Manually tested locally
- [X] Additional issue created in [docs repo](https://github.com/architus/docs.archit.us/issues) (if necessary)

### CODE REVIEW

File: shard/Dockerfile [hunk 1]
```diff
@@ -41,18 +41,7 @@ RUN python3.8 -m pip install wheel
 RUN python3.8 -m pip install --trusted-host pypi.python.org -r requirements.txt
 #RUN apk del libpng-dev jpeg-dev libffi-dev libxml2-dev python3-dev libxslt-dev freetype-dev
 
-WORKDIR /pyre2
```
Comment (johnyburd) [inline shard/Dockerfile:44]: uh
don't we still use this?

Comment (johnyburd) [reply shard/Dockerfile:44]: or was that just a remnant from a previous era that's not doing anything

Comment (jammiess) [reply shard/Dockerfile:44]: I have no clue why that's there. All of pyre2 is set up and installed earlier in the Dockerfile.

Comment (johnyburd) [reply shard/Dockerfile:44]: nice

**Review by johnyburd — APPROVED** [review]

**Pull request merged**